profiles/esha98.json

```
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/60232135/215237852-c021e76d-316d-4fa0-96dd-59d9666bc6fe.png",
+    "issueId": 1687,
+    "name": "Esha",
+    "username": "Esha98"
+  }
+  
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```